server/FakeToOriginalIPConverter/FakeToOriginalIPConverter.js

```diff
@@ -92,7 +92,7 @@ async function modifyServer() {
             let application = applicationList[index];
             let applicationName = application['component'];
```

Comment:
As discussed, in fakeToOriginalIPMapping.json file, let us change the use of category to contain either server/client values, so in code we can include a logic to find whether the instance is meant to update local-address/remote-address

Reply:
Another idea to include new variable "instance-type"